css/style.css

```diff
@@ -287,9 +287,14 @@
 
 /* Описание стилей Третьей и Четвертой секции */
 
+/* Описание третьей секции */
+
 .wrap-information {
   position: relative;
-}
+  padding-bottom: 720px;
+  max-width: 1440px;
+  background-: url(img/jpg/2013-tesla-model-s-cockpit.jpg) 0 0/contain no-repeat;
+  }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)